src/clarity-angular/datagrid/datagrid.ts

```diff
@@ -35,7 +39,10 @@ import {DatagridRenderOrganizer} from "./render/render-organizer";
 export class Datagrid implements AfterContentInit, AfterViewInit, OnDestroy {
 
     constructor(public selection: Selection, private sort: Sort, private filters: FiltersProvider,
-                private page: Page, public rowActionService: RowActionService, public items: Items) {}
+                private page: Page, public items: Items, public rowActionService: RowActionService) {
+    }
+
+    public hasActionableRow = false;
```

Comment:
do you need this? looks like the template is using rowActionService.hasActionableRow directly?